grpc/android-cubic/sdk/src/main/java/cobaltspeech/cubic/CubicGrpc.java

```diff
@@ -0,0 +1,518 @@
+import static io.grpc.stub.ClientCalls.blockingServerStreamingCall;
+import static io.grpc.stub.ClientCalls.blockingUnaryCall;
+import static io.grpc.stub.ClientCalls.futureUnaryCall;
+import static io.grpc.stub.ServerCalls.asyncBidiStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncClientStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncServerStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncUnaryCall;
+import static io.grpc.stub.ServerCalls.asyncUnimplementedStreamingCall;
+import static io.grpc.stub.ServerCalls.asyncUnimplementedUnaryCall;
+
+/**
+ * <pre>
+ * Service that implements the Cobalt Cubic Speech Recognition API
+ * </pre>
+ */
+@javax.annotation.Generated(
+    value = "by gRPC proto compiler (version 1.24.0)",
+    comments = "Source: cubic.proto")
+public final class CubicGrpc {
+
+  private CubicGrpc() {}
+
+  public static final String SERVICE_NAME = "cobaltspeech.cubic.Cubic";
+
+  // Static method descriptors that strictly reflect the proto.
+  private static volatile io.grpc.MethodDescriptor<com.google.protobuf.Empty,
+      cobaltspeech.cubic.CubicOuterClass.VersionResponse> getVersionMethod;
+
+  @io.grpc.stub.annotations.RpcMethod(
+      fullMethodName = SERVICE_NAME + '/' + "Version",
+      requestType = com.google.protobuf.Empty.class,
+      responseType = cobaltspeech.cubic.CubicOuterClass.VersionResponse.class,
+      methodType = io.grpc.MethodDescriptor.MethodType.UNARY)
+  public static io.grpc.MethodDescriptor<com.google.protobuf.Empty,
+      cobaltspeech.cubic.CubicOuterClass.VersionResponse> getVersionMethod() {
+    io.grpc.MethodDescriptor<com.google.protobuf.Empty, cobaltspeech.cubic.CubicOuterClass.VersionResponse> getVersionMethod;
+    if ((getVersionMethod = CubicGrpc.getVersionMethod) == null) {
+      synchronized (CubicGrpc.class) {
+        if ((getVersionMethod = CubicGrpc.getVersionMethod) == null) {
+          CubicGrpc.getVersionMethod = getVersionMethod =
+              io.grpc.MethodDescriptor.<com.google.protobuf.Empty, cobaltspeech.cubic.CubicOuterClass.VersionResponse>newBuilder()
+              .setType(io.grpc.MethodDescriptor.MethodType.UNARY)
+              .setFullMethodName(generateFullMethodName(SERVICE_NAME, "Version"))
+              .setSampledToLocalTracing(true)
+              .setRequestMarshaller(io.grpc.protobuf.lite.ProtoLiteUtils.marshaller(
+                  com.google.protobuf.Empty.getDefaultInstance()))
+              .setResponseMarshaller(io.grpc.protobuf.lite.ProtoLiteUtils.marshaller(
+                  cobaltspeech.cubic.CubicOuterClass.VersionResponse.getDefaultInstance()))
+              .build();
+        }
+      }
+    }
+    return getVersionMethod;
+  }
+
+  private static volatile io.grpc.MethodDescriptor<cobaltspeech.cubic.CubicOuterClass.ListModelsRequest,
+      cobaltspeech.cubic.CubicOuterClass.ListModelsResponse> getListModelsMethod;
+
+  @io.grpc.stub.annotations.RpcMethod(
```

Comment:
It looks like there are some indentation issues with the method below

Reply:
It is generated code